egui/src/align.rs

```diff
@@ -29,6 +29,14 @@ impl Align {
     pub fn bottom() -> Self {
         Self::Max
     }
+
+    pub(crate) fn scroll_center_factor(&self) -> f32 {
+        match self {
+            Self::Min => 0.0,
+            Self::Center => 0.5,
+            Self::Max => 1.0,
+        }
+    }
```

Comment:
I don't know if this is the best place to add this.

Reply:
This is the right place! Maybe could be called simply `to_factor`